frontend/src/routes/password.change.index.tsx

```diff
@@ -86,7 +86,10 @@ export const Route = createFileRoute("/password/change/")({
   component: ChangePassword,
 });
 
-const usePasswordComplexity = (password: string): PasswordComplexity => {
+const usePasswordComplexity = (
+  password: string,
+  t: TFunction<"frontend", undefined>,
+): PasswordComplexity => {
```

Comment:
As this one is already a hook, it's fine to `useTranslation` in the hook body here if you want, it won't cost more to call that multiple times for the same component